### ISSUE TITLE
Dotenv should not die when file `.env` is not present. 

### ISSUE THREAD

**Issue by mikkoi** [book/Dotenv#2]
Using `.env` file should be voluntary. There are many ways to generate the environment. We should not force the user to use .env files.

Instead of dying, the library could emit a warning. It should also be possible to disable this warning (e.g. when scripting).

Would you accept a pull request?

**Comment by szabgab** [comment]
Yeah, I just bumped into the same issue, and I agree with @mikkoi

**Comment by whosgonna** [comment]
FWIW, same thing here.   Is the author open to a PR?

**Comment by marghidanu** [comment]
Same same.

**Comment by bbrtj** [comment]
Bump. I'm willing to help - implement, release or both.
